### DISCUSSION TITLE
Disable default keymaps

### DISCUSSION THREAD

**Discussion by caenrique** [nvim-neo-tree/neo-tree.nvim#1345] [General]
Hi! I just updated to v3.x and now the keymaps seem to be behaving differently. Not sure if this is because new kemaps were added or the override behaviour is different.

In my case, I like using `o` to open directories/files, and new there are a buch of kemaps starting with `o` such as `os`, `og`, etc. Those keymaps make the ux slower for me, because I have a delay when only pressing `o`. 

I have the following config for the window and filesystem provider, which is the only one I'm using:
```lua
window = {
  width = 50,
  mappings = {
    ['s'] = 'open_vsplit',
    ['S'] = 'open_split',
    ['o'] = 'open',
  },
}
```
```lua
filesystem = {
  ...
  window = {
    mappings = {
      ['s'] = 'open_vsplit',
      ['S'] = 'open_split',
      ['o'] = 'open',
    },
  }
}
```

is there a way to disable the default mappings or override them?

**Comment by pysan3** [comment]
[key] = "noop",

The `o` key was not mapped to anything in the previous version, so it was a feature addition and not a breaking change.

You'd either want to disable all `o*` mappings in [defaults.lua](https://github.com/nvim-neo-tree/neo-tree.nvim/blob/cfe1920c5dfb0524b3a13e827c35b6eb571143aa/lua/neo-tree/defaults.lua#L425-L432), or use a different key to open.
It's just a matter of preference but I think many people use `l` or `<CR>` to open file.

**Reply by caenrique** [reply]
> [key] = "noop"

That's what I was looking for, thanks
